### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by iqrammm** [Nico31415/Drowning-Detector#1]
Can you share the dataset you used for training and validation?

**Comment by Nico31415** [comment]
The dataset has already been trained by YOLO. The file yolov3.cfg just contains the weights and biases, while the file yolov3.txt just contains the labels of each classified object.

You can read more about this in https://pjreddie.com/darknet/yolo/

**Comment by suneet2404** [comment]
Nico- Please share the link of dataset. Link provided in previous message it is related to YOLO not about the dataset. Thanks

**Comment by ajithvcoder** [comment]
@Nico31415  could you provide the dataset link please

**Comment by abdulazadh** [comment]
Hi, It would be helpful you provide the dataset.. 
Thanks in Advance

**Comment by waseemshariff126** [comment]
Hi guys, Yolo-V3 is very famous for its pre-trained model. You do not require any dataset as such. Its simple, just use weights and configuration file. If still you require something to train more efficiently, you could use transfer learning.

**Comment by waseemshariff126** [comment]
Hi, could you please provide the testing data. The video were a person is enacting to be drowning.

**Comment by dhruvgoel99** [comment]
yeah please provide data for this

**Comment by yister0321** [comment]
yeah, we need your test video. My e-mail is 464151490@qq.com.  Thanks.

**Comment by KamatVarun** [comment]
> The dataset has already been trained by YOLO. The file yolov3.cfg just contains the weights and biases, while the file yolov3.txt just contains the labels of each classified object.
> 
> You can read more about this in https://pjreddie.com/darknet/yolo/

I understand you have already trained the model. Could you please share the dataset as well and how you created it ?

**Comment by Rishab-Vanigotha** [comment]
> provided in previous message it is related to YOLO not about the datase

https://github.com/sswarnak77/LifeGuard.IO/tree/master/VideoDataset/DataSets/UCF11%20and%20OUR%20OWN%20DATASET/action_youtube_naudio
check out this in this drowning dataset is already collected and stored... drowning and not drowning both are available
but the video clarity is very less
